Assets/GothicVR/Scripts/Creator/VobCreator.cs

```diff
@@ -591,23 +599,155 @@ private static GameObject CreateDecal(PxVobData vob)
+
+            go.name = vob.visualName;
+            go.transform.SetPositionAndRotation(vob.position.ToUnityVector(), vob.rotation.ToUnityMatrix().rotation);
+            go.SetParent(parent);
+
+            var pfx = AssetCache.TryGetPfxData(vob.visualName);
+            var particleSystem = go.AddComponent<ParticleSystem>();
+
+            go.AddComponent<VobPfxProperties>().pfxData = pfx;
+            
+            particleSystem.Stop();
+
+            var gravity = pfx.flyGravity.Split();
+            float gravityX = 1f, gravityY = 1f, gravityZ = 1f;
+            if (gravity.Length == 3)
+            {
+                // Gravity seems too low. Therefore *10k.
+                gravityX = float.Parse(gravity[0], CultureInfo.InvariantCulture) * 10000;
+                gravityY = float.Parse(gravity[1], CultureInfo.InvariantCulture) * 10000;
+                gravityZ = float.Parse(gravity[2], CultureInfo.InvariantCulture) * 10000;
+            }
+            
+            // Main module
+            {
+                var mainModule = particleSystem.main;
+                var minLifeTime = (pfx.lspPartAvg - pfx.lspPartVar) / 1000; // I assume we need to change milliseconds to seconds.
+                var maxLifeTime = (pfx.lspPartAvg + pfx.lspPartVar) / 1000;
+                mainModule.duration = 1f; // I assume pfx data wants a cycle being 1 second long.
+                mainModule.startLifetime = new (minLifeTime, maxLifeTime);
+                mainModule.loop = pfx.ppsIsLooping;
+
+                // I couldn't find a specific property. Therefore using the combined gravity value. ƪ(˘⌣˘)ʃ
+                var speed = 1f;
+                speed *= Math.Abs(gravityX) > 0.01f ? Math.Abs(gravityX) : 1;
+                speed *= Math.Abs(gravityY) > 0.01f ? Math.Abs(gravityY) : 1;
+                speed *= Math.Abs(gravityZ) > 0.01f ? Math.Abs(gravityZ) : 1;
+                
+                var minSpeed = (pfx.velAvg - pfx.velVar) / 1000;
+                var maxSpeed = (pfx.velAvg + pfx.velVar) / 1000;
+                mainModule.startSpeed = new(minSpeed, maxSpeed);
+            }
+
+            // Emission module
+            {
+                var emissionModule = particleSystem.emission;
+                emissionModule.rateOverTime = new ParticleSystem.MinMaxCurve(pfx.ppsValue);
+            }
+
+            // Force over Lifetime module
+            {
+                var forceModule = particleSystem.forceOverLifetime;
+                if (gravity.Length == 3)
+                {
+                    forceModule.enabled = true;
+                    forceModule.x = gravityX;
+                    forceModule.y = gravityY;
+                    forceModule.z = gravityZ;
+                }
+            }
+
+            // Renderer module
+            {
+                var rendererModule = go.GetComponent<ParticleSystemRenderer>();
+                TextureManager.I.SetTexture(pfx.visName, rendererModule.material);
```

Comment:
Default selected shader for the material is URP/Lit which takes a ton of time to compile (even if it's one time only)
Maybe we can change it do Simple Lit or another shader which doesn'y take that much to compile
```suggestion
                var standardShader = Shader.Find("Universal Render Pipeline/Simple Lit");
                var material = new Material(standardShader);
                rendererModule.material = material;
                TextureManager.I.SetTexture(pfx.visName, rendererModule.material);
```